cli/tests/functional/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project>
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.helidon.build-tools.cli.tests</groupId>
+        <artifactId>helidon-cli-tests</artifactId>
+        <version>3.0.0-SNAPSHOT</version>
+    </parent>
+
+    <groupId>io.helidon.build-tools.cli.tests.functional</groupId>
+    <artifactId>helidon-cli-functional</artifactId>
+    <version>3.0.0-SNAPSHOT</version>
+    <name>Helidon Command Line Interface Functional Tests</name>
+
+    <properties>
+        <jarCliPath>${project.basedir}/../../impl/target/helidon.jar</jarCliPath>
```

Comment:
This module does not depend on `cli/impl` so there is no guaranty that `${project.basedir}/../../impl/target/helidon.jar` exists when building this module.

You can use the dependency plugin and make a local distribution of the cli, or you can invoke the cli main class programmatically.